deepinv/training_utils.py

```diff
@@ -527,3 +527,52 @@ def test(
         wandb.log({"Test PSNR": test_psnr}, step=step)
 
     return test_psnr, test_std_psnr, linear_psnr, linear_std_psnr
+
+
+def train_normalizing_flow(
+    model,
+    dataloader,
+    epochs=10,
+    learning_rate=1e-3,
+    device="cpu",
+    jittering=1 / 255.0,
+    verbose=False,
+):
+    """
+    Trains a normalizing flow.
+
+    Uses the forward KL (maximum likelihood) loss function and the Adam optimizer.
```

Comment:
Maybe add 1 sentence and 1 equation stating a bit more explicitely the training objective?